### PR TITLE
chore: remove trailing slash code and docs references

### PR DESCRIPTION
**Pull request by rudouglas**

### CODE REVIEW

File: packages/gatsby-theme-newrelic/README.md [hunk 1]
```diff
@@ -68,17 +67,16 @@ websites](https://opensource.newrelic.com).
     - [`NavItem`](#navitem)
       - [`Page`](#page)
     - [`NewRelicLogo`](#newreliclogo)
-    - [`Overlay`](#overlay)
+    - [Overlay](#overlay)
```
Comment (josephgregoryii) [inline packages/gatsby-theme-newrelic/README.md:70]: was the removal of backticks intentional?

File: packages/gatsby-theme-newrelic/README.md [hunk 2]
```diff
@@ -68,17 +67,16 @@ websites](https://opensource.newrelic.com).
...
     - [`PageTools`](#pagetools)
       - [`PageTools.Section`](#pagetoolssection)
       - [`PageTools.Title`](#pagetoolstitle)
-      - [`Portal](#portal)
+    - [Portal](#portal)
```
Comment (josephgregoryii) [inline packages/gatsby-theme-newrelic/README.md:74]: same question as above

**Review by josephgregoryii — APPROVED** [review]
Looks great! didnt find much needed revision, only a question regarding the `README.md`

File: packages/gatsby-theme-newrelic/README.md (hunk 2 above)
Comment (rudouglas) [reply packages/gatsby-theme-newrelic/README.md:74]: Oh that's a good shout @josephgregoryii they should be present as the other table of contents items have them 😸

**Pull request merged**